pycontw/pycontw-frontend
# fix(financial-aid): refine financial aid page

## Types of changes


* **Bugfix**

## Description

- rename Acceptance Deadline to Deadline, Application How-Tos to Application Process, Some Tips on Application to Some Tips
- fix the content of Auxiliary Items and Reimbursement
- fix table layout
- remove visa part

## Steps to Test This Pull Request

Steps to reproduce the behavior:
1. Go to `/registration/financial-aid` and check the update.

## Expected behavior

![localhost_3000_2021_en-us_registration_financial-aid (2)](https://user-images.githubusercontent.com/18432820/164915342-b23a1f27-39e5-48ad-a608-d0ca9178951d.png)

![localhost_3000_2021_en-us_registration_financial-aid (3)](https://user-images.githubusercontent.com/18432820/164915364-d487bf16-2c42-4ca5-abc4-f77afe0c48ba.png)

<img src=https://user-images.githubusercontent.com/18432820/164915209-4607bf66-0f84-4944-a380-7c7834481e09.png width=300 />
<img src=https://user-images.githubusercontent.com/18432820/164915231-b925ffb6-9b48-41f3-9027-e081c7698ba9.png width=300 />
